crates/bevy_ecs/src/system/commands.rs

```diff
@@ -228,17 +79,11 @@ impl Commands {
     ///     // Create a new entity with two components.
     ///     commands.spawn((Component1, Component2));
     /// }
-    ///
-    /// # example_system.system();
```

Comment:
Without this it is possible that it becomes impossible to turn `example_system` into a system as the necessary signature changes like here.

Reply:
hmm i have no clue why i removed this. maybe to check why a test was failing and I forgot to re-add it? regardless: nice catch.